### DISCUSSION TITLE
2013/11/10/airway-beacon-25-near-johnson-city-tn

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#15] [Announcements]
# 2013/11/10/airway-beacon-25-near-johnson-city-tn

http://localhost:4000/2013/11/10/airway-beacon-25-near-johnson-city-tn

**Comment by jengalas** [comment]
Posted by Shari
November 17, 2013 at 2:26 am
_Imported from WordPress_

Great stuff. Thank you (both) for sharing.

(Found my way here via the article on the arrows/beacons at messynessychic.com. Your lists of all the beacons are brilliant!)
